docs/en/latest/how-to-build.md

```diff
@@ -185,23 +185,29 @@ Run the `apisix help` command to see the returned results and get commands and d
 
 ```shell
 # more actions find by `help`
-$ apisix help
+apisix help
 ```
 
 ## Step 4: Run Test Cases
 
-1. Install `cpanminus`, the package manager for `perl`.
+1. Install `cpanminus`, the package manager for `perl`and dependencies.
```

Comment:
```suggestion
1. Install `cpanminus`, the package manager for `perl` and dependencies.
```

Reply:
Remove ‘and dependencies’, because there is no installation package for this step.